src/Infiltrator.jl

```diff
@@ -111,19 +111,49 @@ end
 Wraps expression in a try block, infiltrate if an exception is raised.
 Equivalent to:
 
+```julia
```

Comment:
Don't need the indentation here if you add the fence.

Would also be good to add the `rethrow`s in both cases.